linera-chain/src/lib.rs

```diff
@@ -145,6 +145,8 @@ pub enum ChainError {
 #[derive(Copy, Clone, Debug)]
 pub enum ChainExecutionContext {
     Query,
+    #[cfg(with_testing)]
+    ReadBytecodeLocation,
```

Comment:
nit: Can we put this one at the end? (So that the variant tags are stable)

Reply:
Done.